torch/_ops.py

```diff
@@ -15,6 +16,9 @@
 from torch._functorch.pyfunctorch import dispatch_functorch
 from torch.utils._python_dispatch import TorchDispatchMode
 
+_T = TypeVar("_T")
+_P = ParamSpec("_P")
+
 
 _F = TypeVar("_F", bound=Callable[..., Any])
```

Comment:
Get rid of `F_`, as it can still cause problems. Another PR is open to improve the situation

Reply:
It seems like #144047 fixes this (existing) issue - so I'll land this and let that PR fix it.